velox/functions/prestosql/BingTileFunctions.h

```diff
@@ -39,9 +53,10 @@ struct BingTileFunction {
       return Status::UserError(
           fmt::format("Bing tile Y coordinate {} cannot be negative", y));
     }
-    if (FOLLY_UNLIKELY(zoom < 0)) {
-      return Status::UserError(
-          fmt::format("Bing tile zoom {} cannot be negative", zoom));
+
+    auto zoomCheck = checkBingTileZoom(zoom);
+    if (FOLLY_UNLIKELY(!zoomCheck.ok())) {
+      return std::move(zoomCheck);
```

Comment:
This is causing the fuzzer builds to fail.
https://github.com/facebookincubator/velox/actions/workflows/scheduled.yml